Documentation/policy/language.rst

```diff
@@ -1124,7 +1124,7 @@ is labeled correctly in your environment. In the example configuration, you can
 run ``kubectl get nodes -o wide|grep type=ingress-worker`` to verify labels
 match the policy.
 
-You can verify the policy was properly parsed and applied by ``exec``ing into
+You can verify the policy was applied by running ``kubectl exec`` in
```

Comment:
This new wording can be read as "run kubectl inside the cilium agent pod", which doesn't make sense since kubectl isn't available there. Is it important for us to even mention execing or pods here or just say "Run cilium policy get..."?

If we think this should still be here, maybe we can clarify the wording a bit more, such as:

```suggestion
You can verify the policy was applied by running ``kubectl exec -n $CILIUM_NAMESPACE cilium-xxxx cilium policy get`` for the cilium pod ... 
```

Reply:
I changed the wording to be more clear per your suggestion but I'm open to making further changes. I was intending to assume no prior knowledge so I included the exec command in full in your example. Let me know if I should revert to something simpler.